test/e2e/framework/test_context.go

```diff
@@ -196,6 +201,7 @@ func RegisterCommonFlags() {
 func RegisterClusterFlags() {
 	flag.BoolVar(&TestContext.VerifyServiceAccount, "e2e-verify-service-account", true, "If true tests will verify the service account before running.")
 	flag.StringVar(&TestContext.KubeConfig, clientcmd.RecommendedConfigPathFlag, os.Getenv(clientcmd.RecommendedConfigPathEnvVar), "Path to kubeconfig containing embedded authinfo.")
+	flag.StringVar(&TestContext.ExternalKubeConfig, fmt.Sprintf("%s-%s", clientcmd.RecommendedConfigPathFlag, "external"), clientcmd.RecommendedHomeFile, "Path to kubeconfig containing embedded authinfo for external cluster. Valid only for kubemark")
```

Comment:
You probably can remove "Valid only for kubemark" line as this possibly can be used even without kubemark (e.g if you want to use a custom kubeconfig). Or maybe just say something like "Used for talking to kubemark master".